react/src/components/search-app/SearchResultsComponent.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+
+export default class SearchResultsComponent extends React.Component {
```

Comment:
Hmm, interesting. Then the page will get a touch of a SPA.

My idea was to separate the components as far as possible so that they can easily be included as widgets in other sites. Moreover, the SearchBarComponent is currently only on the home page but could maybe also included at the top of every other page. Then, I'm not sure if your approach would always work. What do you think? (Also @martomi? @Binabiko?)